content/blog/2022/12/2022-12-09-jfrog-maintenance.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Maintenance with downtime of JFrog Artifactory (repo.jenkins-ci.org) 18 of December of 2022"
+tags:
+- jenkins
+- jfrog
+- artifactory
+- repo.jenkins-ci.org
+- infra
+authors:
+- dduportal
+// opengraph:
+//   image: /images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png
+links:
+  discourse: true
+---
+
+== Maintenance with downtime of JFrog Artifactory (repo.jenkins-ci.org) December 18, 2022
+
+image::/images/sponsors/jfrog.png[JFrog, width=20%, align=center, link="https://jfrog.com/"]
+
+December 18, 2022: our sponsor link:https://jfrog.com/[JFrog] will proceed to the maintenance of our "jenkinsci" Artifactory instance.
+
+Expect a complete downtime of about 6 hours due to the nature of this maintenance: a cloud migration from Google Cloud to Amazon Web Services.
+
+Jenkins users won't be impacted: plugin downloads, update center and websites such as https://jenkins.io and https://plugins.jenkins.io will continue working as expected.
+Only Jenkins contributors will be impacted because no releases or builds on ci.jenkins.io or on developer machines will suceed during the downtime.
+
+This migration is a strong requirement to ensure that JFrog is able to monitor the service and provide the best availability and infrastructure health.
+
+We'll update the link:https://status.jenkins.io[Jenkins Infrastructure Status Page] with the exact downtime window.
+
+== Artifactory Service repo.jenkins-ci.org: a backbone of the Jenkins project
+
+Despite not being directly used by the usual Jenkins users (link:https://www.jenkins.io/download/mirrors/[Jenkins Download Mirrors are used across the world]),
+the https://repo.jenkins-ci.org service is central for any contributor to the Jenkins project: it hosts all the development dependencies and released binaries as the source of truth.
```

Comment:
```suggestion
Despite not being directly used by the usual Jenkins users, link:https://www.jenkins.io/download/mirrors/[Jenkins Download Mirrors are used across the world].
The link:https://repo.jenkins-ci.org[repo.jenkins-ci.org] service is crucial for any contributor to the Jenkins project as it hosts all the development dependencies and released binaries as the source of truth.
```
small suggestion for wording/clarity

Reply:
🤔 I think my original sentence is misleading. Your suggestion does not work alas.

The parentesis `(link:https://www.jenkins.io/download/mirrors/[Jenkins Download Mirrors are used across the world])` implies that the usual Jenkins users are using mirrors across the world but not the repo.jenkins-ci.org service.



WDYT of the simplified version below?

```suggestion
Despite not being directly used by the usual Jenkins users, the link:https://repo.jenkins-ci.org[repo.jenkins-ci.org] service is crucial for any contributor to the Jenkins project as it hosts all the development dependencies and released binaries as the source of truth.
```